_config.yml

```diff
@@ -92,3 +92,9 @@ authors:
     email: luis@ombulabs.com
     twitter: bronzdoc
     github: bronzdoc
+  abizzinotto:
+    display_name: "Amanda Bizzinotto"
+    gravatar:
```

Comment:
@ABizzinotto I just added this changes. Can you put your gravatar id (https://en.gravatar.com/) here so you have an avatar like the rest of us?

Reply:
@ABizzinotto Once you address this comment this will be ready to go  🚀

Reply:
@lubc addressed :) Sorry, I had missed this comment.